### PR TITLE
chore: add volunteer Palakkgoyal

### PR DESCRIPTION
**Pull request by Palakkgoyal**
I want to volunteer with you and help other developers and expand my knowledge

### CODE REVIEW

File: data/users/Palakkgoyal.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Palak Goyal",
+  "bio": "Frontend Developer",
+  "image": "https://avatars.githubusercontent.com/u/116902573?v=4",
+  "categories": [
+    {
+      "slug": "web-development",
+      "topics": ["CSS", "JavaScript", "React", "Tailwind"]
```
Comment (codewithvoid) [inline data/users/Palakkgoyal.json:8]: Mention JavaScript in separate `programming` category. 

**Example**
```json
{
  "slug": "programming",
  "topics": ["JavaScript"]
}
```

**Review by codewithvoid — CHANGES_REQUESTED** [review]
Thanks for volunteering Palak. 

Added a small comment. Please check.

**Comment by Palakkgoyal** [comment]
I have done the changes. Please check

**Review by codewithvoid — APPROVED** [review]
Perfect. Welcome to the community. 

[Discord Link](https://discord.gg/DFwsFKfgEB)

**Comment by Palakkgoyal** [comment]
Thanks a lot Void

**Pull request merged**